src/components/molecules/SpaceEditForm/SpaceEditForm.tsx

```diff
@@ -331,11 +335,11 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({
             >
               <SpacesDropdown
                 venueSpaces={backButtonOptionList ?? ALWAYS_EMPTY_ARRAY}
-                venueId={venueId}
                 setValue={setValue}
                 register={register}
                 fieldName="venue.parentId"
-                defaultSpace={venueValues.parentId}
+                defaultSpace={parentSpace}
+                error={errors?.venue?.parentId}
```

Comment:
[OOS]
This should  be refactored to how `AdminInput` and `AdminCheckbox` deal with `errors`.